scripts/json-schema/gen-schema.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs');
 
 const program = TJS.getProgramFromFiles([
     resolve(root + '/src/lib/content.d.ts'),
-    resolve(root + '/src/index.d.ts'),
+    resolve(root + '/index.d.ts'),
```

Comment:
£1 kerching

Not now, but this could use my new and exciting `urlJoin` function.

Reply:
![tenor-236670507](https://user-images.githubusercontent.com/638051/72622637-8a0a3580-393b-11ea-9985-23209c4fe105.gif)